packages/react/src/patterns/sub-patterns/PictogramItem/PictogramItem.js

```diff
@@ -19,27 +20,27 @@ const { prefix } = settings;
  * @param {object} props props object {title, copy, pictogram, link}
  * @param {string} props.heading Content with pictogram component title property
  * @param {string} props.copy Content with pictogram component copy property
+ * @param {string} props.className Pictogram Item className prop
  * @param {*} props.Pictogram Content with pictogram component Pictogram component
  * @returns {*} Content array with pictograms item JSX Component
  */
-const PictogramItem = ({ heading, copy, Pictogram, cta }) => (
-  <div
-    data-autoid={`${stablePrefix}--pictogram-item`}
-    className={`${prefix}--pictogram-item`}>
+const PictogramItem = ({ heading, copy, Pictogram, cta, className }) => (
+  <div className={classNames(className, `${prefix}--pictogram-item`)}>
     <div className={`${prefix}--pictogram-item__row`}>
-      <div className={`${prefix}--pictogram-item__pic-col`}>
-        <Pictogram
-          data-autoid={`${stablePrefix}--pictogram-item_pictogram`}
+      <div className={`${prefix}--pictogram-item__wrapper`}>
+        <Pictogram.src
+          data-autoid={`${stablePrefix}--pictogram-item__pictogram`}
           className={`${prefix}--pictogram-item__pictogram`}
+          {...Pictogram}
```

Comment:
I think you might be passing in src unnecessarily into the pictogram as an attribute.

You might do something like the following...

```jsx
const PictogramItem = ({ heading, copy, pictogram: {src: Pictogram, ...pictogramProps}, cta, className }) => (
  <Pictogram
    {...pictogramProps}
```